tests/crypto/mbedtls/prj.conf

```diff
@@ -1,4 +1,4 @@
-CONFIG_ZTEST_STACKSIZE=4096
+CONFIG_ZTEST_STACKSIZE=8192
```

Comment:
a little worried how this will impact other targets.

If this is solely needed for RISC-V, could it be done in testcase.yml, as:
```
    extra_configs:
      - CONFIG_ZTEST_STACKSIZE=8192
    arch_allow: riscv32
```

Reply:
This has been updated according to your recommendation in the latest force-push. Thank you.

Reply:
@martin-aberg thanks a lot.

it turned out that my hunch was correct, at least I found the `stm32f3_disco` board to fail when increasing stack size.
With stack size of: `4096`.
```
Memory region         Used Size  Region Size  %age Used
           FLASH:      130522 B       256 KB     49.79%
            SRAM:       29494 B        40 KB     72.01%
        IDT_LIST:         120 B         2 KB      5.86%
```

With stack size of: `8192`.
```
 region `SRAM' overflowed by 822 bytes
```

and twister actual informs of this but doesn't fail the build, it just skips it.
```
            "test_details":[
                {
                    "testcase":"crypto.mbedtls.mbedtls",
                    "arch":"arm",
                    "type":"integration",
                    "platform":"stm32f3_disco",
                    "status":"skipped",
                    "reason":"SRAM overflow"
                },
```